src/resources/AzPowerShellSdkStrings.Designer.cs

```diff
@@ -61,7 +61,7 @@ internal class AzPowerShellSdkStrings {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Could not add additional information with name {name} to binding {bindingName} because no binding with this name was found.
```

Comment:
In the `resources` folder we should only have this file: `/resources/AzPowerShellSdkStrings.resx`. What is the other one used for?

Reply:
Just a file autocreated by Visual Studio. I removed it.